src/acrcssc/azext_acrcssc/_params.py

```diff
@@ -0,0 +1,38 @@
+        c.argument("schedule", options_list=["--schedule"], help=f"schedule to run the scan and patching task. E.g. `<n>d` where <n> is the number of days between each run. Max value is {CONTINUOUSPATCH_SCHEDULE_MAX_DAYS}d.", required=True)
+        c.argument("run_immediately", options_list=["--run-immediately"], help="Set this flag to trigger the immediate run of the selected workflow task. Default value: false.", arg_type=get_three_state_flag(), required=False)
+        c.argument("dryrun", options_list=["--dry-run"], help="Use this flag to see the qualifying repositories and tags that would be affected by the workflow. Default value: false. 'config' parameter is mandatory to provide with dry-run", arg_type=get_three_state_flag(), required=False)
+
+    with self.argument_context("acr supply-chain workflow update") as c:
+        c.argument("config", options_list=["--config"], help="Configuration file path containing the json schema for the list of repositories and tags to filter within the registry. Schema example:{\"repositories\":[{\"repository\":\"alpine\",\"tags\":[\"tag1\",\"tag2\"],\"enabled\":true},{\"repository\":\"python\",\"tags\":[\"*\"],\"enabled\":false}], \"version\": \"v1\", \"tag-convention\": \"floating\"}}. \"tag-convention\" is an optional property, values can be \"incremental\" (the default behavior, will increase the patch version of the tag, for example \"{repository}:{original-tag}-1\", \"{repository}:{original-tag}-2\", etc), or \"floating\" (will reuse the tag \"{repository}:{original-tag}-patched\" for patching)", required=False)
+        c.argument("schedule", options_list=["--schedule"], help=f"schedule to run the scan and patching task. E.g. `<n>d` where n is the number of days between each run. Max value is {CONTINUOUSPATCH_SCHEDULE_MAX_DAYS}d.", required=False)
+        c.argument("run_immediately", options_list=["--run-immediately"], help="Set this flag to trigger the immediate run of the selected workflow task. Default value: false.", arg_type=get_three_state_flag(), required=False)
+        c.argument("dryrun", options_list=["--dry-run"], help="Use this flag to see the qualifying repositories and tags that would be affected by the workflow. Default value: false. 'config' parameter is mandatory to provide with dry-run", arg_type=get_three_state_flag(), required=False)
+
+    with self.argument_context("acr supply-chain workflow list") as c:
+        c.argument("status", arg_type=get_enum_type(WorkflowTaskState), options_list=["--run-status"], help="Status to filter the supply-chain workflow image status.", required=False)
+
+    with self.argument_context("acr supply-chain workflow delete") as c:
+        c.argument("yes", options_list=["--yes", "-y"], help="Proceed with the deletion without user confirmation", required=False)
```

Comment:
```suggestion
    with self.argument_context("acr supply-chain workflow list") as c:
        c.argument("status", arg_type=get_enum_type(WorkflowTaskState), help="Status to filter the supply-chain workflow image status.")

    with self.argument_context("acr supply-chain workflow delete") as c:
        c.argument("yes", options_list=["--yes", "-y"], help="Proceed with the deletion without user confirmation")
```

Reply:
not sure if a bug on cli core, but removing the `required=False` on `--run-status` marks it as required. The other ones are fine
![image](https://github.com/user-attachments/assets/1810cada-7be3-4294-981a-2d3b6c1b24a7)

Reply:
similar to `--config` on `update` command

Reply:
also didn't remove the `options_list` on the parameters that change the signature (i.e dryrun -> dry-run)